frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/OverviewSettings.tsx

```diff
@@ -86,18 +88,20 @@ export const OverviewSettings = ({
             setSelectedClientCodeOptionName={setSelectedClientCodeOptionName}
           />
 
-          <Box my="1rem">
-            <Center>
-              <h4>{jt`More ${(
-                <ExternalLink
-                  key="examples"
-                  href="https://github.com/metabase/embedding-reference-apps"
-                >
-                  {t`examples on GitHub`}
-                </ExternalLink>
-              )}`}</h4>
-            </Center>
-          </Box>
+          {showMetabaseLinks && (
```

Comment:
I'm not actually sure about hiding the example links as in the project of hiding Metabase doc links, we only talk about the `/learn/*` and `/docs/*` links.

I asked a question [in Slack here.](https://metaboat.slack.com/archives/C063Q3F1HPF/p1706080479826559)